mypyc/test-data/irbuild-tuple.test

```diff
@@ -258,3 +258,233 @@ L14:
     r11 = r15
 L15:
     return r11
+
+
+[case testTupleBuiltFromList]
```

Comment:
Also add a test where we execute the code, in `mypyc/test-data/run-tuples.test`. You can add more sub-cases to the existing `testTupleOps` test. Note that it uses the default driver, so you can just add `test_<whatever>` function(s). Test iterating over a list, a tuple and an unsupported iterable (e.g. set).

Reply:
Thanks @JukkaL ! I just add a test function including iterating over a `list`, a `tuple` and a `RTuple`(unsupported).